src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -1324,7 +1324,6 @@ namespace HighTempRadiantSystem {
         // Initialize arrays
         state.dataHeatBalFanSys->SumConvHTRadSys = 0.0;
         state.dataHeatBalFanSys->SumLatentHTRadSys = 0.0;
-        state.dataHeatBalFanSys->QHTRadSysSurf = 0.0;
         state.dataHeatBalFanSys->QHTRadSysToPerson = 0.0;
```

Comment:
So, here's a place where previously `QHTRadSysSurf` and `QHTRadSysToPerson` were reset to zero on every call to `DistributeHTRadGains`.
(Why is `QHTRadSysToPerson` still in this branch, by the way?)
 is called twice in [`CalcHighTempRadiantSystemSP`](https://github.com/NREL/EnergyPlus/blob/82391071e9a009712653ee6017118dce65dacf90/src/EnergyPlus/HighTempRadiantSystem.cc#L970). And again in `UpdateHighTempRadiantSystem`. This will accumulate HT rad gains onto `QHTRadSysSurf` multiple times before they get zeroed again. I'm surprised there are no diffs. It's possible that `CalcHighTempRadiantSystemSP` isn't covered? No, integration coverage shows it's used. What am I missing? How is this working without diffs?